bin/firefox-driver.js

```diff
@@ -43,4 +51,21 @@ if (shouldStart) {
   setInterval(() => {}, 100);
 }
 
+function getResults(driver) {
+  driver
+    .findElement(By.id("mocha-stats"))
+    .getText().then(results => {
+      console.log("results ", results);
+      const match = results.match(/failures: (\d*)/);
+      const resultCode = parseInt(match[1], 10) > 0 ? 1 : 0;
+      process.exit(resultCode);
+    });
+}
+
+if (isTests) {
+  const driver = start();
+  driver.get("http://localhost:8003");
+  setTimeout(() => getResults(driver), 5000);
```

Comment:
I think the canonical way to do it is to introduce a `setInterval` that checks for something and then carries out an action. So we could check the progress element for 100%, and then get the report. Honestly, I'd rather not do that work now as it's also somewhat painful to write and I'd rather bump this to 30 seconds if we need to at some point. 5 seconds is already 5x longer than it takes 😉  If it fails, it'll be because a test stalls and we need to bail anyways.